src/Illuminate/Collections/Collection.php

```diff
@@ -701,7 +701,7 @@ public function last(callable $callback = null, $default = null)
     /**
      * Get the values of a given key.
      *
-     * @param  string|array<array-key, string>  $value
+     * @param  string|int|array<array-key, string>  $value
```

Comment:
Not sure if this can also be replaced with `array-key` which should stand for `string|int` but has a better semantic representation
```suggestion
     * @param  array-key|array<array-key, string>  $value
```

Reply:
I'm not entirely sure you're supposed to use array-key like that. Empirically it seems to work as you intend.